Tests/Optiq.DatePicker.Tests/DatePicker/DatePickerUnitTests.cs

```diff
@@ -0,0 +1,210 @@
+﻿using Xunit;
+using Optiq.DatePicker;
+
+namespace Optiq.DatePicker.Tests;
+
+public class DatePickerUnitTests
+{
+    [Fact]
+    public void TestMinimumDate()
+    {
+        Optiq.DatePicker.DatePicker picker = new Optiq.DatePicker.DatePicker();
+
+        picker.MinimumDate = new DateTime(1950, 1, 1);
+
+        Assert.Equal(new DateTime(1950, 1, 1), picker.MinimumDate);
+
+        
+        Assert.Throws<ArgumentException>(() => picker.MinimumDate = new DateTime(2200, 1, 1));
+    }
+
+
+    [Fact]
+    public void TestNullableDate()
+    {
+        Optiq.DatePicker.DatePicker picker = new Optiq.DatePicker.DatePicker();
+        if (picker.Date == null)
+        {
+            Assert.Null(picker.Date);
+        }
+
+    }
+
+    [Fact]
+    public void TestMaximumDate()
+    {
+        Optiq.DatePicker.DatePicker picker = new Optiq.DatePicker.DatePicker();
+
+        picker.MaximumDate = new DateTime(2050, 1, 1);
+
+        Assert.Equal(new DateTime(2050, 1, 1), picker.MaximumDate);
+
+        Assert.Throws<ArgumentException>(() => picker.MaximumDate = new DateTime(1800, 1, 1));
```

Comment:
shouldn't you set the "picker.Date" to an invalid value causing an exception?
Or what are you actually testing here?

Reply:
testing that the maximum value isnt valid when its smaller than the minimum date